tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -60,7 +60,7 @@ struct Buffer {
 void parse_single_object(RawDataPtr ptr, json const& j, MetaComponent const& meta, Idx position) {
     for (auto const& it : j.items()) {
         // Allow and skip unknown attributes
-        if (!meta.has_attribute(it.key())) {
+        if (static_cast<int>(meta.has_attribute(it.key())) == 0) {
```

Comment:
```suggestion
        if (static_cast<Idx>(meta.has_attribute(it.key())) == 0) {
```

in addition: why do you need to cast to `int`? isn't the data type already `Idx` (which is `int`-type)

Reply:
Yes, that's true, it was a suggestion by clang tidy, let's see if it accepts it your way